scippy/test/test_plotting.py

```diff
@@ -1,146 +0,0 @@
-# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
I don't think this should be removed. @nvaytet just excluded it from the tests that run by default, iirc?

Reply:
It looked to me like some of the function calls had gone stale. There were also some changes around the `scippy/__init__.py` that made me think this was no longer necessary. I'll discuss with @nvaytet